frontend/src/components/pages/Scheduling/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { NavigationProps, Step, useForm, useStep } from "react-hooks-helper";
+
+import ConfirmDetails from "./ConfirmDetails";
+import DonationInformation from "./DonationInformation";
+import GetStarted from "./GetStarted";
+import SelectDateTime from "./SelectDateTime";
+import VolunteerInformation from "./VolunteerInformation";
+
+const steps = [
+  {
+    id: "get started scheduling",
+  },
+  {
+    id: "date and time",
+  },
+  {
+    id: "donation information",
+  },
+  {
+    id: "volunteer information",
+  },
+  {
+    id: "confirm donation details",
+  },
+];
+
+interface UseStepType {
+  step: number | Step | any;
+  navigation: NavigationProps | any;
+}
+
+const Scheduling = () => {
+  const [formValues, setForm] = useForm({
+    category: "",
+    quantity: 0,
+    size: "",
+    pickupLocation: "",
+    startTime: new Date().toDateString(),
```

Comment:
@hanlinc27 Are we changing the type for startTime and endTime to string for frontend?

Reply:
changed it to string for now